### ISSUE TITLE
[docs] Provide a way to order content in TOCs

### ISSUE THREAD

**Issue by imphil** [lowRISC/opentitan#3464]
Currently, all content in the TOCs of the documentation is ordered in more or less (to the reader) random ways. We need a way to manually specify the ordering of content, e.g. to have the user guide start with the "getting started" documentation.

I did play around with the weight front matter attribute, but it doesn't do anything in my experiments. Hence over to the Hugo pro, aka @gkelly.

**Comment by rswarbrick** [comment]
I don't think this is true any more, and it probably changed quite a bit with the move to mdbook.
